### ISSUE TITLE
Not working on PS 1.6.1.18

### ISSUE THREAD

**Issue by Alexezi** [faktiva/prestashop-clean-urls#98]
not working on 1.6.1.18 , at least not for me. I worked fine on 1.6.1.12 , then I made an upgrade. Before upgrade I turned uninstalled the module.

Then I install back on again  ( latest version 1.2.3 ) and whatever I do ..clear cash , save  , friendly URL on/off ...all that according the manual ( Ive done it before on last version so haven't missed anything). Still not working. It's not overriding / deleting the ID . webs site still shows the ID numbers in URL

When I try in BO to delete {id}_   ...it returns as an error when trying to save it." Keyword "{id}" required ..."

I have tried to uninstall and complete delete folder via FTP and then install back again from scratch...still nothing :-(

**Comment by drAlberT** [comment]
looks like the module install hasn't gone right ..   you could have something usefull in your PHP logs

**Comment by feliperuizweb** [comment]
I had the same problem with version 1.6.1.18 at the moment I will save the rules of url the prestashop displayed the error alert saying "Keyword "{id}" required for route"

My php log did not generate any errors, and I did the installation of the module according to the documentation here in Git

**Comment by Alexezi** [comment]
I uninstalled the module., delete it completely out.
Advance Parameters - Performance section :  cleared cash , allow override (debug mode ) , recompile templates if file has been changed  . 
upload back the module.
When tried to install it gave an error saying its there already so cleared cash again, then install OK.
After all this I can freely rewrite lines in Routes. (URL&SEO section ) and goes green .
Basically Ive found out that almost after any step u do, keep clear the cash in BO ( not the one in browser ) .
Takes time and nerves. Hope that helps.

**Comment by feliperuizweb** [comment]
I followed what was said above did the cache cleaning but what came out effect was disable the option that disables all the overrides, after that worked perfectly!

**Comment by drAlberT** [comment]
great job guys .. so the issue is that PS update was disabling overrides ???

**Comment by Alexezi** [comment]
yep, seem like.
The disabling overrides  and clear cash is better be done even before the installation of the module.
